test/absinthe/schema/sdl_render_test.exs

```diff
@@ -0,0 +1,223 @@
+defmodule SdlRenderTest do
+  use ExUnit.Case
+
+  @moduledoc """
+  https://wehavefaces.net/graphql-shorthand-notation-cheatsheet-17cd715861b6
+  https://github.com/graphql/graphql-js/blob/master/src/utilities/schemaPrinter.js
+
+  TODO:
+    - [ ] Directives on lots of things? like types & fields
```

Comment:
If this is still TODO, pull out to issue

Reply:
https://github.com/absinthe-graphql/absinthe/issues/766